pontoon/insights/static/js/insights_charts.js

```diff
@@ -40,6 +40,60 @@ var Pontoon = (function (my) {
         Chart.defaults.global.datasets.bar.barPercentage = 0.7;
         Chart.defaults.global.datasets.bar.categoryPercentage = 0.7;
       },
+      // Legend configuration doesn't allow for enough flexibility,
+      // so we build our own legend
+      // eslint-disable-next-line no-unused-vars
+      customLegend: function (chart) {
+        return function (chart) {
+          function renderLabels(chart) {
+            return chart.data.datasets
```

Comment:
This is why we have lambdas. Also, a code comment would be Really Nice for why this wrapping is required.

```suggestion
      customLegend: (chart) => (chart) => (chart) => chart.data.datasets
```

Reply:
BTW, this block was also copied from elsewhere.